code_folder/A1_csv_to_json.py

```diff
@@ -0,0 +1,356 @@
+                "output_lp_file",
+                "overwrite",
+                "path_input_file",
+                "path_input_folder",
+                "path_output_folder",
+                "path_output_folder_inputs",
+                "restore_from_oemof_file",
+                "start_date",
+                "store_oemof_results",
+                "timestep",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="simulation_settings", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "project_data.csv":
+            parameters = [
+                "country",
+                "label",
+                "latitude",
+                "longitude",
+                "project_id",
+                "project_name",
+                "scenario_id",
+                "scenario_name",
+                "sectors",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="project_data", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif f == "economic_data.csv":
+            parameters = [
+                "currency",
+                "discount_factor",
+                "label",
+                "project_duration",
+                "tax",
+            ]
+            single_dict = create_json_from_csv(
+                input_directory, filename="economic_data", parameters=parameters
+            )
+            input_json.update(single_dict)
+        elif "storage_" in f:
+            pass
+        else:
+            logging.error(
+                "The file %s" % f + " is not recognized as input file for mvs "
+                "check /mvs_eland/inputs/elements/default_csv for correct "
+                "file names."
+            )
+
+    with open(os.path.join(input_directory, output_filename), "w") as outfile:
+        json.dump(input_json, outfile, skipkeys=True, sort_keys=True, indent=4)
+    logging.info("json stored into /mvs_eland/inputs/%s", output_filename)
+    if pass_back:
+        return input_json
+
+
+def create_json_from_csv(input_directory, filename, parameters):
+
+    """
+    One csv file is loaded and it"s parameters are checked. The csv file is
+    then converted to a dictionary; the name of the csv file is used as the
+    main key of the dictionary. Exceptions are made for the files
+    ["economic_data", "project", "project_data", "simulation_settings"], here
+    no main key is added. Another exception is made for the file
+    "energyStorage". When this file is processed, the according "storage_"
+    files (names of the "storage_"-columns in "energyStorage" are called and
+    added to the energyStorage Dictionary.
+
+
+    :param input_directory: str
+        directory name in which the csv files are stored in
+    :param filename: str
+        name of the file that is transformed into a json, without ending
```

Comment:
Is it an output filename?
```suggestion
        name of the file that is transformed into a json, without extension
```

Reply:
it's the input file. Changed this.